### PR TITLE
Use improved table names.

### PR DESCRIPTION
**Pull request by deflaux**
There are really only two changes in here:

  1. "the expanded table" conveys no information, I renamed it to get across what it holds, see https://github.com/googlegenomics/codelabs/commit/a8698f32107487c77b63ab12b61b505a136d9229
  2. also renamed "gender" to "sex" because it is the more common term for anatomical sex at birth, see https://github.com/googlegenomics/codelabs/commit/3615fe46b196198c5cbb8284f5688701337005a4

### CODE REVIEW

File: R/PlatinumGenomes-QC/Data-Transformation.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 In Part 2 of the codelab, we perform a data transformation to make it more amenable to our QC analyses in Parts 3 and 4 of this codelab.
 
+We refer to the original table as the "genome calls" table.  It contains all reference calls and variant calls.  To facilitate variant-centric analysis, we generate a second table, the "multi-sample variants" table, based on the genome calls to document calls for all the variable sites. These are the positions where a variant was detected in any of the genomes in the cohort. Positions where only reference calls were detected were omitted, hence the the "multi-sample variants" table resembles the commonly accepted multi-sample VCF file.
```
Comment (mbookman) [inline R/PlatinumGenomes-QC/Data-Transformation.md]: I had a hard time understanding this paragraph. How about the following:

We refer to the original table as the "genome calls" table. It contains *all* reference calls and variant calls. To facilitate variant-centric analysis, we generate a second table, the "multi-sample variants" table.

The multi-sample variants table resembles a multi-sample VCF file. In this table:

* every variant record includes calls for *all* callsets
* variants which contained *only reference calls for all callsets* are omitted

File: R/PlatinumGenomes-QC/Data-Overview.Rmd [hunk 2]
```diff
@@ -43,7 +43,8 @@ source("./rHelpers/setup.R")
 ```{r}
 # By default this codelab runs upon the Illumina Platinum Genomes Variants.  
 # Change the table here if you wish to run these queries against your own data.
-queryReplacements <- list("_THE_TABLE_"="genomics-public-data:platinum_genomes.variants")
+queryReplacements <- list("_THE_TABLE_"="genomics-public-data:platinum_genomes.variants",
+                          "_GENOME_CALL_TABLE_"="genomics-public-data:platinum_genomes.variants")
```
Comment (mbookman) [inline R/PlatinumGenomes-QC/Data-Overview.Rmd]: Should one of these be _MULTISAMPLE_VARIANT_TABLE_?

File: R/PlatinumGenomes-QC/comparison/QC-Comparison.md [hunk 3]
```diff
@@ -466,38 +469,38 @@ joinedResult <- inner_join(expectedResult, result, by=c("INDV"))
...
+The logic of the query is the same as vcftools [output_het method](http://sourceforge.net/p/vcftools/code/HEAD/tree/trunk/cpp/variant_file_output.cpp#l165), but the filtering criteria are slightly different, which explains the small difference in the results. The primary difference is due to using only SNPs in the query whereas vcftools uses both SNPs and indels. This is because the "expanded table" used in the query only contains non-variant segments for SNPs (i.e. indels matching the reference are not included), which results in inaccurate statistics for indels. There are a few other minor differences as well. For instance, vcftools discards the entire record if a [single sample is non-diploid](http://sourceforge.net/p/vcftools/code/HEAD/tree/trunk/cpp/entry_getters.cpp#l94) whereas the query only discards non-diploid samples within a record. Overall, the filtering differences are minor and the output of the query is largely consistent with vcftools.
```
Comment (mbookman) [inline R/PlatinumGenomes-QC/comparison/QC-Comparison.md]: I think this paragraph should be broken up into three:

The logic of the query ...

The primary difference ...

There are a few other ...

File: R/PlatinumGenomes-QC/Data-Transformation.md (hunk 1 above)
Comment (deflaux) [inline R/PlatinumGenomes-QC/Data-Transformation.md]: fixed!

File: R/PlatinumGenomes-QC/Data-Overview.Rmd (hunk 2 above)
Comment (deflaux) [inline R/PlatinumGenomes-QC/Data-Overview.Rmd]: no, but I simplified the code, it was unnecessarily confusing

File: R/PlatinumGenomes-QC/comparison/QC-Comparison.md (hunk 3 above)
Comment (deflaux) [inline R/PlatinumGenomes-QC/comparison/QC-Comparison.md]: fixed

**Comment by deflaux** [comment]
thanks @mbookman !  PTAL

**Comment by mbookman** [comment]
LGTM

**Pull request merged**